mh/session.py

```diff
@@ -0,0 +1,163 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Monster Hunter session module.
+
+    Monster Hunter 3 Server Project
+    Copyright (C) 2021  Sepalani
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import mh.database as db
+import mh.pat_item as pati
+
+from other.utils import to_bytearray, to_str
+
+DB = db.get_instance()
+
+
+class Session(object):
+    """Server session class.
+
+    TODO:
+     - Finish the implementation
+    """
+    def __init__(self):
+        """Create a session object."""
+        self.local_info = {
```

Comment:
Why create a dictionary for this variables? Instead of just plain vars?

Reply:
I was too lazy to create a `LocalInfo` class that'd represent what the game calls "Local Info" in the online UI. It probably wouldn't be directly stored directly in the session but rather would be a link to another table when I was thinking in a SQL/database way.

Reply:
That sound good